pilot/cmd/pilot-discovery/main.go

```diff
@@ -98,8 +98,8 @@ func hasKubeRegistry() bool {
 func init() {
 	discoveryCmd.PersistentFlags().StringSliceVar(&serverArgs.Service.Registries, "registries",
 		[]string{string(serviceregistry.KubernetesRegistry)},
-		fmt.Sprintf("Comma separated list of platform service registries to read from (choose one or more from {%s, %s, %s, %s})",
```

Comment:
Would it make sense to mention if nothing specified the default is MCP? since in the case of Cloud Foundry you run with  `--registries ""` to enable MCP. Also wondering if that works by eliminating `--registries` all together. E.g only provide `--registries` if we want k8s, consult, etc when nothing provided it would take MCP by default.

Reply:
yeah, i'd prefer to have `--registries` be optional and have MCP the default.

Reply:
/hold

we can't change the defaults just because they're not used by someone. 

You can override it in your platform.